content/strategy-goals/strategy/code-graph/search/core.md

```diff
@@ -1,38 +1,46 @@
 
 ## What's next and why
 
-### FY23Q1
+### FY23Q2
 
 #### Goals
 
-- **Monorepo performance**: At a P75 level, gigarepo will index in less than 30 minutes, indexed searches complete in < 2s and unindexed searches complete in < 10s. Gigarepo is a representative monorepo with a HEAD working copy size of 15GB.
-- **Ranking**: We start tracking ranking quality, using selected search results as a proxy.
-- **Code host coverage**: Sourcegraph Cloud indexes public repositories globally from the most popular package hosts.
+- **Monorepo performance**: At a P75 level, synthetic gigarepo will index in less than 30 minutes, indexed searches will complete in < 2s and unindexed searches will complete in < 10s.
+- **Ranking**: Understand customer pain points and establish a baseline for future improvements.
+- **Dependencies Search**: Expand support for dependencies search to Python and JVM and implement `repo:dependents(...)`.
 
 #### Details
 
-**Monorepo performance**: is a recurrent pain point for large enterprise customers. Having replicated large monorepo setups, we identified that unindexed monorepo performance is still poor and several facets of search on large monorepos cause significant load on gitserver.
+**Monorepo performance**: Is a recurrent pain point for large enterprise customers. Having replicated large monorepo setups, we identified that unindexed monorepo performance is still poor and several facets of search on large monorepos cause significant load on gitserver. To address this, we have been working on an incremental indexing prototype and a Searcher prototype to better handle unindexed searches.
 
-**Ranking**: As a first step towards improving the ranking of our search results, we will start tracking the quality of search results using the index of user-selected results as a proxy metric. Having this tracking in place will help measure the success of future improvements and drive the areas of ranking we choose to focus on.
+**Ranking**: As a first step towards improving the ranking of our search results, we will start tracking the quality of search results using the index of user-selected results as a proxy metric. Having this tracking in place will help measure the success of future improvements. We will also conduct discovery of customers needs and pain points to drive the areas of ranking we choose to focus on.
 
-**Code host coverage**: as we seek to keep expanding the code we index to include more non-GitHub.com or GitLab.com code hosts, we will add support for package host integrations (PyPi, Rubygems, NuGet, Crates, proxy.golang.org). This will not only increase our code host coverage, but also be a stepping stone towards unblocking [use cases based on the dependency graph of repositories](https://docs.google.com/document/d/1SkM8CG0IksvPEKRBRVLKipiRJTopx6Vq_hSWRJ9NyKs/edit#).
```

Comment:
Are we completely dropping package host integrations. I know we have fedora set up, are we not planning to pursue the rest?

Reply:
We are getting a lot of that via dependency search.